common/src/main/resources/META-INF/native-image/io.netty/common/native-image.properties

```diff
@@ -12,4 +12,6 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-Args = --initialize-at-run-time=io.netty.util.AbstractReferenceCounted,io.netty.util.concurrent.GlobalEventExecutor,io.netty.util.concurrent.ImmediateEventExecutor,io.netty.util.concurrent.ScheduledFutureTask,io.netty.util.internal.ThreadLocalRandom
+Args = --initialize-at-run-time=io.netty.util.AbstractReferenceCounted,io.netty.util.concurrent.GlobalEventExecutor,io.netty.util.concurrent.ImmediateEventExecutor,io.netty.util.concurrent.ScheduledFutureTask,io.netty.util.internal.ThreadLocalRandom,io.netty.util.internal.logging.Log4JLogger \
+       -Dio.netty.noUnsafe=true \
+       -Dio.netty.leakDetection.level=DISABLED
```

Comment:
why this one ? I think this is not what we want...

Reply:
:+1: I'd let the final user decide about the `-D` flags